src/Mabiavalon.DockNC/Docking/Branch.cs

```diff
@@ -135,7 +151,7 @@ private void UpdateLogicalChildren(object oldItem, object newItem)
 
             if (oldChild != null)
             {
-                ((ISetLogicalParent) oldItem).SetParent(null);
+                //((ISetLogicalParent)oldItem).SetParent(null);
```

Comment:
Could you explain this change? Wouldn't this cause issues with the logical tree?